src/databricks/labs/ucx/install.py

```diff
@@ -130,6 +130,8 @@ def __init__(
         if not environ:
             environ = dict(os.environ.items())
         self._force_install = environ.get("UCX_FORCE_INSTALL")
+        # TODO: add a similar env parameter for force_dependecies_download
+        self._force_dependencies = environ.get("UCX_FORCE_DEPENDENCIES") == "True"
```

Comment:
1. Use prompts.confirm, environment variables are hardly discoverable
2. Add this property to WorkspaceConfig instead of leaking it through constructors, it might be useful for updating as well.